docs/reference/snapshot-restore/apis/get-snapshot-status-api.asciidoc

```diff
@@ -0,0 +1,273 @@
+
+`<repository>`::
+(Required, string)
+Snapshot repository name used to limit the request. Wildcard (`*`) expressions are supported.
++
+To retrieve a list of all snapshots in the cluster, omit this and the `<snapshot>` parameter.
+
+`<snapshot>`::
+(Required, string)
+Comma-separated list of snapshot names to retrieve status for.
++
+To retrieve a list of all snapshots in a specified repository, omit this parameter.
++
+NOTE: Wildcard (`*`) expressions are not supported for `<snapshot>`.
+
+[role="child_attributes"]
+[[get-snapshot-status-api-request-body]]
+==== {api-request-body-title}
+
+`ignore_unavailable`::
+(Optional, boolean)
+If `false`, the request returns an error for any snapshots that are unavailable. Defaults to `false`.
++
+If `true`, the request ignores snapshots that are unavailable, such as those that are corrupted or temporarily cannot be returned.
+
+[role="child_attributes"]
+[[get-snapshot-status-api-response-body]]
+==== {api-response-body-title}
+`repository`::
+(string)
+Name of the repository that includes the snapshot.
+
+`snapshot`::
+(string)
+Name of the snapshot.
+
+`uuid`::
+(string)
+Universally unique identifier (UUID) of the snapshot.
+
+`state`::
++
+--
+(string)
+Indicates the current snapshot state.
+
+.Values for `state`
+[%collapsible%open]
+====
+`FAILED`::
+  The snapshot finished with an error and failed to store any data.
+
+`IN_PROGRESS`::
+  The snapshot is currently running.
+
+`PARTIAL`::
+  The global cluster state was stored, but data of at least one shard was not stored successfully.
+  The <<get-snapshot-api-response-failures,`failures`>> section of the response contains more detailed information about shards
+  that were not processed correctly.
+
+`SUCCESS`::
+  The snapshot finished and all shards were stored successfully.
+====
+--
+
+`include_global_state`::
+(boolean)
+Indicates whether the current cluster state is included in the snapshot.
+
+[[get-snapshot-status-shards-stats]]
+`shards_stats`::
+(object)
+Contains a count of shards in the snapshot.
++
+.Properties of `shards_stats`
+[%collapsible%open]
+====
+`initializing`::
+(integer)
+Number of shards that are still initializing.
+
+`started`::
+(integer)
+Number of shards that have started but not are not finalized.
+
+`finalizing`::
+(integer)
+Number of shards that are finalizing but are not done.
+
+`done`::
+(integer)
+Number of shards that initialized, started, and finalized successfully.
+
+`failed`::
+(integer)
+Number of shards that failed to be included in the snapshot.
+
+`total`::
+(integer)
+Total number of shards included in the snapshot.
+====
+
+[[get-snapshot-status-stats]]
+`stats`::
+(object)
+Provides details on the number (`file_count`) and size (`size_in_bytes`) of files included in the snapshot.
++
+.Properties of `stats`
+[%collapsible%open]
+====
+`incremental`::
+(object)
+Number and size of files that are not already in the repository and need to be copied as part of the incremental snapshot.
```

Comment:
This needs to be worded a little differently. If you run this API for an already complete snapshot then it would be 

>  Number and size of files that were not already in the repository and needed to be copied as part of the incremental snapshot. 

I think we should explain both cases here in the wording somehow.

Reply:
I think that we can amend the original statement to focus on the files that still need to be copied, versus the delta of files that didn't already exist and need copying:

>Number and size of files that still need to be copied as part of the incremental snapshot.

>For completed snapshots, this property indicates the number and size 
>of files that were not already in the repository and were copied as 
>part of the incremental snapshot.